build/illume.js

```diff
@@ -11,17 +11,15 @@ var _q = require("@standard-library/q");
 var _kefir = require("kefir");
 
 var offsetAbove = function offsetAbove(y, element) {
-  return y > $(element).offset().top;
+  return y > element.offsetTop;
```

Comment:
Not sure about browser support, but the `Element.getBoundingClientRect()` seems like the method with the least amount of support among the different methods in the jQuery Code, and basic support is available Chrome (1.0 and up), Edge, Firefox (3.0 and up), IE (4.0 and up), Opera, and Safari (4.0 and up) for that method. 

I can add a function. Am not sure how would I go about testing this or confirming that it works?